platform-tests/src/test/java/org/junit/platform/engine/support/descriptor/FilePositionTests.java

```diff
@@ -43,13 +55,42 @@ void filePositionFromLine() throws Exception {
 
 	@Test
 	@DisplayName("create FilePosition from factory method with line number and column number")
-	void filePositionFromLineAndColumn() throws Exception {
+	void filePositionFromLineAndColumn() {
 		FilePosition filePosition = FilePosition.from(42, 99);
 
 		assertThat(filePosition.getLine()).isEqualTo(42);
 		assertThat(filePosition.getColumn()).contains(99);
 	}
 
+	@ParameterizedTest
+	@MethodSource
+	void filePositionFromQuery(String query, int expectedLine, int expectedColumn) {
+		Optional<FilePosition> optionalFilePosition = FilePosition.fromQuery(query);
+		if (optionalFilePosition.isPresent()) {
+			FilePosition filePosition = optionalFilePosition.get();
+
+			assertThat(filePosition.getLine()).isEqualTo(expectedLine);
+			assertThat(filePosition.getColumn().orElse(-1)).isEqualTo(expectedColumn);
+			return;
+		}
+
+		assertEquals(-1, expectedLine);
+		assertEquals(-1, expectedColumn);
+	}
+
+	@SuppressWarnings("unused")
+	static Stream<Arguments> filePositionFromQuery() {
+		return Stream.of( //
+			Arguments.of(null, -1, -1), //
```

Comment:
`Arguments.of` --> `arguments` (via static import 😉 )

Reply:
Sure. 🙂